vocode/streaming/agent/token_utils.py

```diff
@@ -57,6 +57,7 @@
     "gpt-4-turbo": 127940,
     "gpt-4o": 127940,
     "gpt-4o-2024-05-13": 127940,
+    "gpt-4o-2024-08-06": 127940,
```

Comment:
could we make this a prefix to all `gpt-4o` models? can ignore if this isn't quick